__tests__/app.test.js

```diff
@@ -268,3 +269,35 @@ describe('/api/articles/:article_id/comments', () => {
       })
     })
   })
+
+  describe('DELETE /api/comments/:comment_id', () => {
+    test('DELETE /api/comments/:comment_id returns a 204 status code ', () => {
+      return request(app)
+      .delete('/api/comments/1')
+      .expect(204)
+    })
+    test('Returns 404 when comment doesnt exist', () => {
+      return request(app)
+      .delete('/api/comments/999')
+      .expect(404)
+      .then((res) => {
+        expect(res.body.msg).toBe('COMMENT DOES NOT EXIST!')
+      })
+    })
+    test('Returns 400 when invalid data type used', () => {
+      return request(app)
+      .delete('/api/comments/test')
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe('INVALID INPUT')
+      })
+    })
+    test('Returns 404 when incorrect url used', () => {
```

Comment:
No need to include a bad path test in this endpoints describe block as this won't run through the functionality for this endpoint. We may have already dealt with all bad paths anyway, did this test auto pass?